<launch>
    <include file="$(find feature-detect)/launch/usb_cam.launch"/>

    <include file="$(find darknet_ros)/launch/darknet_ros.launch"/>

    <node name="feature_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/darknet_ros/detection_image"/>
        <param name="autosize" value="true" />
    </node>

    
</launch>